### ISSUE TITLE
Automatically anchor cables

### ISSUE THREAD

**Issue by Technici4n** [AppliedEnergistics/Applied-Energistics-2#6983]
Gimp doesn't like floating cables, but I like them. We should make it easy for players to place lots of cables with anchors connecting to walls if they want to. (Opt in or opt out, we choose).

It would be nice to not have to craft these anchors.

**Comment by Technici4n** [comment]
Yeah you're right. We need to check what it would look like first with the current model before making it automatic.

**Comment by mindforger** [comment]
to add, i would not like anchors on every single part of cable even if just one side per cable ...  just a few visually appealing ones like i do it currently with the item version here and there

**Comment by SirOlrik** [comment]
Wouldn't it be easier, to use a wrench and disconnect the link to other cables.
I like the way Mekanism does it, if you accidently wrenched/configured a connection out, you just wrench/configure the side, that should connect again, pretty awesome and very easy to maintain, also more aesthetic to look at compared to tons of Anchors.

**Comment by ViciousVinnyD** [comment]
I think a good temp solution would be for cables to place anchors on touching sides while anchors are in the offhand. Similar to how create places blocks under the track from the offhand.

**Comment by 62832** [comment]
Just one thing that hasn't been brought up yet... if we remove Cable Anchors as an item to craft, how do we go about making Facades then?

**Comment by Kevin-Marsh** [comment]
I would suggest crafting the block with a quartz knife to make facades. It works as a mechanic since you are effectively cutting the block into pieces. I would also suggest making the craft knife + block = 4 facades.

**Comment by Mithi83** [comment]
Why 4 facades? A block has 6 faces. If you cut that into pieces I'd expect 6 facades. I know the current recipe does that but I never understood why from a lore perspective. From a game mechanic perspective the recipe would look weird with 6 cable anchors, but that argument would be obsolete with the new approach.
